props/saber_BC_buttons.h

```diff
@@ -1847,7 +1845,9 @@ class SaberBCButtons : public PROP_INHERIT_PREFIX PropBase {
   void ToggleSpamBlast() {
     spam_blast_ = !spam_blast_;
     PVLOG_NORMAL << (spam_blast_ ? "** Entering" : "** Exiting") << " Spam Blast Mode\n";
-    if (!hybrid_font.PlayPolyphonic(&SFX_mzoom)) {
+    if (SFX_mzoom) {
```

Comment:
mzoom is required, so you don't really need to check if it exists or not
(Which means you can also remove EFFECT(mzoom))

Reply:
Even better. And yes, I had removed EFFECT(mzoom).
done.